content/en/docs/concepts/storage/volumes.md

```diff
@@ -922,17 +922,14 @@ receive updates for those volume sources.
 
 ### quobyte
 
-A `quobyte` volume allows an existing [Quobyte](https://www.quobyte.com) volume to
-be mounted into your Pod.
-
-{{< note >}}
-You must have your own Quobyte setup and running with the volumes
-created before you can use it.
-{{< /note >}}
+**Quobyte's in-tree plugin is deprecated.** Use
```

Comment:
Use a ` {{< note >}}` callout here.

Reply:
Hi @venkatsc .
Can you add a 
```
{{< note >}}
...
{{< /note >}}
``` 
shortcode around the deprecation information?